vision/product_search/product_search_add_product_to_product_set.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Google, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vision_product_search_add_product_to_product_set]
+require "google/cloud/vision"
+
+
+def product_search_add_product_to_product_set(
+    project_id     = "your-project-id",
+    location       = "us-west1",
+    product_id     = "your-product-id",
+    product_set_id = "your-product-set-id"
+  )
+  client = Google::Cloud::Vision::ProductSearch.new
+
+  # Get the full path of the product set.
+  product_set_path = client.class.product_set_path(project_id,
```

Comment:
😭😭😭

I know changes to this will be coming across the board for Ruby. Referencing the issue here to share my sadness. No changes required for this PR.

https://github.com/googleapis/gapic-generator-ruby/issues/6